cypress/e2e/tests/administration/jira-connection/negative.test.ts

```diff
@@ -0,0 +1,162 @@
+    const expectedToFail = true;
+    const useTestingAccount = true;
+    const isSecure = false;
+    let jiraBasicCredential: JiraCredentials;
+    let jiraBasicCredentialInvalid: JiraCredentials;
+    let jiraCloudConnectionData: JiraConnectionData;
+    let jiraCloudConnectionDataIncorrect: JiraConnectionData;
+    let jiraCloudConnection: Jira;
+    let jiraCloudConnectionIncorrect: Jira;
+    let applicationList: Array<Assessment> = [];
+    const now = new Date();
+    now.setDate(now.getDate() + 1);
+    const end = new Date(now.getTime());
+    end.setFullYear(end.getFullYear() + 1);
+    let migrationWave: MigrationWave;
+
+    before("Login and create required credentials", function () {
+        // Perform login
+        login();
+        // Defining and creating credentials to be used in further tests
+        jiraBasicCredential = new JiraCredentials(
+            getJiraCredentialData(CredentialType.jiraBasic, useTestingAccount)
+        );
+
+        jiraBasicCredential.create();
+
+        // Defining Jira connection data with correct credentials
+        jiraCloudConnectionData = getJiraConnectionData(
+            jiraBasicCredential,
+            isSecure,
+            useTestingAccount
+        );
+
+        jiraCloudConnection = new Jira(jiraCloudConnectionData);
+
+        // Defining and creating dummy credentials to be used further in tests
+        jiraBasicCredentialInvalid = new JiraCredentials(
+            getJiraCredentialData(CredentialType.jiraBasic, !useTestingAccount)
+        );
+
+        jiraBasicCredentialInvalid.create();
+
+        // Defining Jira connection data with incorrect credentials
+        jiraCloudConnectionDataIncorrect = getJiraConnectionData(
+            jiraBasicCredentialInvalid,
+            isSecure,
+            useTestingAccount
+        );
+
+        jiraCloudConnectionIncorrect = new Jira(jiraCloudConnectionDataIncorrect);
+
+        applicationList = createMultipleApplications(2);
+    });
+
+    beforeEach("Login", function () {
+        cy.intercept("DELETE", "/hub/migrationwaves*/*").as("deleteWave");
+    });
+
+    it("Creating Jira connection with incorrect credentials", () => {
+        jiraCloudConnectionIncorrect.create();
+        jiraCloudConnectionIncorrect.validateState(expectedToFail);
+    });
+
+    it("Removing Jira connection in use by wave", () => {
+        const issueType = JiraIssueTypes.task;
+        let projectName = "";
+        jiraCloudConnection.create();
+        jiraCloudConnection.validateState(!expectedToFail);
+
+        // Defining and creating new migration wave
+        migrationWave = new MigrationWave(
+            data.getRandomWord(8),
+            now,
+            end,
+            null,
+            null,
+            applicationList
+        );
+        migrationWave.create();
+
+        jiraCloudConnection
+            // Getting all projects on Jira side
+            .getAllProjects()
+            .then((project) => {
```

Comment:
This variable should be named "projects" as it is an array

Reply:
Fixed